IDE/STM32Cube/wolfSSL_conf.h

```diff
@@ -1,4 +1,4 @@
-/* wolfSSL_conf.h (example of generated wolfSSL.wolfSSL_conf.h)
+/* wolfSSL_conf.h (example of generated wolfSSL.I-CUBE-wolfSSL_conf.h)
```

Comment:
Does the file name need to be changed from "wolfSSL_conf.h" to "I-CUBE-wolfSSL_conf.h" as well?

Reply:
No, this file is an example only and purposely named different to avoid conflicts. The `I-CUBE-wolfSSL_conf.h` is generated by the STM Cube HAL. There is a note in the README.md and below.